packages/dendron-next-server/components/layout.tsx

```diff
@@ -28,37 +23,37 @@ export default function Layout({
         <meta name="og:title" content={siteTitle} />
         <meta name="twitter:card" content="summary_large_image" />
       </Head>
-      <Header>
-        <div className={styles.logo}>
-          <img
+
+      <Flex height="full" direction="column" data-FUCK>
+        <Flex
+          as="header"
+          bgColor="gray.900"
+          color="white"
+          padding={4}
+          alignItems="center"
+        >
+          <Image
             src="https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/logo-256.png"
-            className={`${utilStyles.borderCircle}`}
             id="logo"
-            style={{ height: "55px" }}
             alt={name}
+            boxSize={16}
           />
-        </div>
-        <Menu theme="dark" mode="horizontal" style={{ float: "right" }}>
-          <Menu.Item key="3">
+
+          <Box marginLeft="auto">
             {" "}
-            <a target="_blank" href="https://dendron.memberful.com/account/subscriptions">Update Subscription</a>
-          </Menu.Item>
-        </Menu>
-      </Header>
+            <Link
+              target="_blank"
+              href="https://dendron.memberful.com/account/subscriptions"
+            >
+              Update Subscription
```

Comment:
appreciate this line. can we have a little icon to notify people that this will take them to an external site?

also, since most people don't have subscriptions, maybe change the text to "Contribute to Dendron 🌱 "

Reply:
:heavy_check_mark: @ e027a6d1